pull-requests.md

```diff
@@ -0,0 +1,162 @@
+Pull Request
+=======
+
+### Why do we use a Pull Request workflow?
+
+PRs are a great way of sharing information, and can help us be aware of the changes that are occurring in our codebase. They are also an excellent way of getting peer review on the work that we do, without the need to work in direct pairs.
+
+**Ultimately though, the primary reason we use PRs is to encourage quality contributions to our code repositories**
+
+Done well, the commits (and their attached messages) contained within tell a story to people examining the code at a later date. If we are not careful to ensure the quality of these commits, we silently lose this ability.
+
+**Poor quality code can be refactored. A terrible commit lasts forever.**
+
+
+### What constitutes a good PR?
+
+A good quality PR will have the following characteristics:
+
+* It will be a complete piece of work that adds value in some way.
+* It will have a title that reflects the work within, and a summary that helps to understand the context of the change.
+* There will be well written commit messages, with well crafted commits that tell the story of the development of this work.
+* Ideally it will be small and easy to understand. Single commit PRs are usually easy to submit, review, and merge.
+* The code contained within will meet the [best practices](./README.md#best-practices) wherever possible.
+
+A PR does not end at submission though. A code change is not made until it is merged and used in production.
+
+A good PR should be able to flow through a peer review system easily and quickly.
+
+
+Submitting Pull Requests
+----
+
+### Ensure there is a human readable title and summary
+
+PRs are a Github workflow tool, so it's important to understand that the PR title, summary and eventual discussion are not as trackable as the the commit history. If we ever move away from Github, we'll likely lose this information.
+
+That said however, they are a very useful aid in ensuring that PRs are handled quickly and effectively.
+
+Ensure that your PR title is scannable. People will read through the list of PRs attached to a repo, and must be able to distinguish between them based on title. Include a ticket/issue reference if possible, so the reviewer can get any extra context. Include a reference to the subsystem affected, if this is a large codebase.
+
+It is recommended to use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) commit structure where appropriate. It provides an easy set of rules for creating an explicit commit history; which makes it easier to write automated tools on top of.
+
+
+### Rebase before you make the PR, if needed
```

Comment:
I think we should make this a single line with a reference to branching / rebasing guidelines?